test/CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ if(EPROSIMA_BUILD_TESTS)
     add_subdirectory(xtypes)
     add_subdirectory(dds/communication)
     add_subdirectory(dds/discovery)
+    add_subdirectory(xmlvalidation)
```

Comment:
It is really not my intention to keep this going back and forth forever, but I just realized this PR is adding tests for the python CLI in a place different from the existing ones. These test should have been added [here](https://github.com/eProsima/Fast-DDS/blob/e31c0075f90e7e6726fbe3d6645fe1131504874b/test/CMakeLists.txt#L76-L79) IMO.